src/index.html

```diff
@@ -6,7 +6,39 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="body">
+    <div class="body__calendar calendar calendar--start-day--default calendar--month-length--default">
```

Comment:
```suggestion
    <div class="calendar calendar--days-count--31 calendar--start-day--thu">
```
you did it wrong, the main task was that if you change the number of days the length of the calendar also changed

Reply:
yes, but it changed.. If i change modificators in code or in developer tools in Mozilla, calendar changes too

Reply:
i just wrote 'default' instead of values because i wanted them to be without exact values